### ISSUE TITLE
[CRITICAL BUG] waydroid /dev/anbox-binder has died HELP!!!

### ISSUE THREAD

**Issue by alihassancods** [waydroid/waydroid#1485]
### Describe the bug

I have already enabled PSI=1 In my kernel parameters but it still doesn't work.
It does work and launched properly after first install and init.
But when I installed Libhoudini using waydroid extras script it doesn't work since then.
Can you help me?

### Waydroid version

1.4.2

### Device

Linux Desktop

### Operating System

Ubuntu 22.04 LTS

### Kernel version

6.5.0-44-generic

### Desktop Environment

Gnome 42.9

### GPU

AMD ATI Radeon HD 5000/6000/735

### Logs

Actually it do not work.
And the logs were kinda this.
Ignoring clip board manager due to missing pyclip package
[gBinder] anbox binder has appeared
[warning] anbox-binder has died.

**Comment by zyxhere** [comment]
Duplicate of https://github.com/waydroid/waydroid/issues/1487 ?

**Comment by ljgermain** [comment]
For me, this was resolved by enabling `CONFIG_COMPAT` in the kernel. Waydroid depends on IA32 emulation to work.

**Comment by unknown20243** [comment]
optiplex@optiplex-OptiPlex-7010:~$ cat /proc/pressure/cpu
some avg10=0.81 avg60=0.91 avg300=1.08 total=154727032
full avg10=0.00 avg60=0.00 avg300=0.00 total=0

psi is enabled by default in latest linux mint (or i copied pasted using chatgpt and it worked) but still i am getting the same error on linux mint saying anbox-binder appeared and then died
i also has
optiplex@optiplex-OptiPlex-7010:~$ grep CONFIG_IA32_EMULATION /boot/config-$(uname -r)
CONFIG_IA32_EMULATION=y
# CONFIG_IA32_EMULATION_DEFAULT_DISABLED is not set
optiplex@optiplex-OptiPlex-7010:~$

**Comment by a3gh** [comment]
@unknown20243 Were you able to resolve the isuse or were you able to gain additional information on the root cause?

**Comment by a3gh** [comment]
> Duplicate of https://github.com/waydroid/waydroid/issues/1487 ?

@zyxhere does not seem to be a duplicate. In #1487 anbox-binder appears and then hangs, while for this issue it explicitly dies

**Comment by unknown20243** [comment]
i found the answer. it is simple. use that tutorial
https://docs.waydro.id/faq/using-custom-waydroid-images
download the same or latest version of binders. but same version of image and binder is nesscessary. and it works. it is that simple.

also English is my 6th language so i can't write easily in it
